Packs/Workday/Integrations/WorkdayIAMEventsGenerator/README.md

```diff
@@ -0,0 +1,78 @@
+Generates Workday dummy test reports and events. Used for testing/dev purposes.
```

Comment:
Generates mock Workday reports and events, which you should use for testing and development purposes.